src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_data_plane_commands.py

```diff
@@ -52,11 +48,12 @@ def test_batch_certificate_cmd(self, resource_group, batch_account_name):
             self.check('thumbprintAlgorithm', 'sha1'),
             self.check('state', 'deleting')])
 
-    # Since this test requires core quota, we must statically create an account with quota
+    @ResourceGroupPreparer()
+    @BatchAccountPreparer()
     def test_batch_pool_cmd(
```

Comment:
Change to use resource preparers and you can use environment variables `AZURE_CLI_TEST_DEV_RESOURCE_GROUP_NAME` and `AZURE_CLI_TEST_DEV_BATCH_ACCT_NAME` for existing resources in live tests when necessary.